src/app/apps/rup/mapa-camas/sidebar/cama-detalle/cama-detalle.component.ts

```diff
@@ -292,4 +293,35 @@ export class CamaDetalleComponent implements OnInit {
         return this.permisosMapaCamasService.indicacionesCrear || this.permisosMapaCamasService.indicacionesEjecutar ||
             this.permisosMapaCamasService.indicacionesValidar || this.permisosMapaCamasService.indicacionesVer;
     }
+
+    setDropDown(relacion, drop) {
+        this.relacionesPosibles = { ...relacion };
+        if (this.openedDropDown) {
```

Comment:
Se podría simplificar este if  por:  this.openedDropDown?.open = this.openedDropDown === drop ? true : false; de este modo te asegurarias de que si openedDropDown existe, sigue con la operacion, en caso contrario, no se realiza.